app/src/test/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityLocalTest.kt

```diff
@@ -103,6 +111,33 @@ class QuestionPlayerActivityLocalTest {
     profileTestHelper.initializeProfiles()
   }
 
+  @Test
+  fun testQuestionPlayer_submitCorrectAnswer_correctTextBannerIsDisplayed() {
+    launchForQuestionPlayer(SKILL_ID_LIST).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.question_recycler_view)).check(matches(isDisplayed()))
+
+      // Submit correct answer
+      submitCorrectAnswerToQuestionPlayerFractionInput()
+
+      onView(withId(R.id.congratulations_text_view))
+        .check(matches(withEffectiveVisibility(VISIBLE)))
```

Comment:
Can we use ``isCompletelyDisplayed()`` instead?

Reply:
Done